export/wiki/species/xp.py

```diff
@@ -19,29 +21,34 @@ class LevelExperienceRampType(Enum):
     MAX = 4
 
 
+OFFICIAL_SERVER_MAX_TAME_LEVEL = 450
+
+
 class LevelData(ExportModel):
-    ramp: str = Field(
+    wildOverride: Optional[IntLike] = None
```

Comment:
Add a description saying a value here means the spawns won't respect difficulty.